.travis.yml

```diff
@@ -1,15 +1,12 @@
-sudo: false
 language: node_js
 node_js:
```

Comment:
the only question is do we want:

```yml
os:
  - linux
  - osx
```

or would that be overkill?

Reply:
I'd say lets add osx.